### PR TITLE
History

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files to check

`src/components/historycard/HistoryCard.jsx`
`src/context/history-context.js`
`src/page/history/History.jsx`
`src/reducer/history-reducer.js`

## Thank you for reviewing

### CODE REVIEW

**Review by arjun-computer-geek — APPROVED** [review]
Code looks cleaner. Nice work @Developers-codz

File: src/page/home/home.module.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .cardContainer {
-  width: 75vw;
+  /* width: 75vw; */
```
Comment (ApurvaSawant11) [inline src/page/home/home.module.css]: You can remove the comment

File: src/reducer/history-reducer.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  switch (type) {
+    case "ADD_IN_HISTORY":
+      return { ...state, historyList: payload };
+
+    case "REMOVE_FROM_HISTORY":
+      return { ...state, historyList: payload };
+
+    case "CLEAR_ALL_HISTORY":
+      return { ...state, historyList: payload };
+
```
Comment (ApurvaSawant11) [inline src/reducer/history-reducer.js:11]: All of these have the same return statements. You can declare only one with a generic name like `SET_HISTORY` and that for all the operations

**Review by ApurvaSawant11 — APPROVED** [review]
The code looks clean and easy to understand. Just had one suggestion

File: src/page/home/home.module.css (hunk 1 above)
Comment (Developers-codz) [reply src/page/home/home.module.css]: Okay

File: src/reducer/history-reducer.js (hunk 2 above)
Comment (Developers-codz) [reply src/reducer/history-reducer.js:11]: To keep the functions pure I have wrote three different cases

**Pull request merged**